multiraft/rpc.proto

```diff
@@ -0,0 +1,45 @@
+// Copyright 2015 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License. See the AUTHORS file
+// for names of contributors.
+//
+// Author: Tamir Duberstein (tamird@gmail.com)
+
+syntax = "proto2";
+package cockroach.multiraft;
+option go_package = "multiraft";
+
+import "github.com/coreos/etcd/raft/raftpb/raft.proto";
```

Comment:
Proto imports operate in a different space than go imports. Importing a proto with a go-style path will impair our ability to use our protos from other languages (etcd's raft proto is incorrect in this respect). This isn't a huge problem for this internal-only proto, but adding `$GOPATH/src` to the import path for all our proto builds will invite mistakes later on. 

I don't think we should import `raftpb/raft.proto` from our own protos. Instead, RaftMessageRequest.message should be a byte string holding an encoded `raftpb.Message`.